### ISSUE TITLE
Move mute/unmute video button to left side if app is in left-handed mode

### ISSUE THREAD

**Issue by altmoola** [QuantumBadger/RedReader#1160]
Straight forward, I think the mute/unmute button should move to the left side if the user has configured the app to be in left-handed mode. This prevents the reach across and accidental screen press when browsing with your left hand.

![image](https://github.com/QuantumBadger/RedReader/assets/15036268/4ad27626-c7f6-4c1b-bb00-aa531dabe27f)![image](https://github.com/QuantumBadger/RedReader/assets/15036268/8295da3c-368b-4f65-a9bc-632c624398c4)

**Comment by QuantumBadger** [comment]
Agreed -- this setting should also affect the floating comment navigation buttons.
